test_ros2cli/package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>test_ros2cli</name>
+  <version>0.7.4</version>
+  <description>
+    Tests for ROS2 command line tools.
+  </description>
+  <maintainer email="ivanpauno@ekumenlabs.com">Ivan Paunovic</maintainer>
+  <license>Apache License 2.0</license>
+
+  <test_depend>action_tutorials</test_depend>
```

Comment:
@ivanpauno Is this an existing package? if yes, where is it?

Currently this rosdep key fails to resolve in the nightly CI jobs: http://build.ros2.org/view/Eci/job/Eci__nightly-fastrtps_ubuntu_bionic_amd64/44/console

Reply:
https://github.com/ros2/demos/tree/master/action_tutorials

Reply:
It was, but was later refactored in `action_tutorials_interfaces`, `action_tutorials_py`, `action_tutorials_cpp` after https://github.com/ros2/demos/pull/378.

Reply:
https://github.com/ros2/ros2cli/issues/310

Reply:
https://github.com/ros2/ros2cli/issues/310